src/clusterfuzz/stacktraces/constants.py

```diff
@@ -638,10 +641,12 @@
     (UBSAN_MISALIGNED_ADDRESS_REGEX, 'Misaligned-address'),
     (UBSAN_NO_RETURN_VALUE_REGEX, 'No-return-value'),
     (UBSAN_NULL_ARGUMENT_REGEX, 'Invalid-null-argument'),
+    (UBSAN_IMPLICIT_CONVERSION_REGEX, 'Implicit-conversion'),
```

Comment:
Does it makes sense to add these to https://github.com/google/clusterfuzz/blob/9cf7ee03c0fc9c427300aad69906d4ea86e060c2/src/clusterfuzz/_internal/crash_analysis/crash_analyzer.py#L82 as well? 

Otherwise these will get marked as security.